[MTM-60126]: Update release notes for certificate chain inferred from device certificate (#2293)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10-20-485-0-X-509-certificate-chain-inferred-from-device-certificate.md

---
date:

> title: Added support for certificate authentication and retrieving JWT session token from device certificate only

Reviewer: ```suggestion
title: Certificate authentication and retrieval of JWT session token from device now supported
```

product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:

>   - value: q3kclF6pO

Reviewer 2: @BeateRixen  is this `1q3kclF6pO` changed to  `component-q3kclF6pO` ? Because in the document it looks updated 
![Uploading image.png…]()

Reviewer: No, it hasn´t changed. I think we planned to do that. I will change it in the document for now.

Reviewer: In the document, it was correct.

    label: Authentication
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-54838
version: 10.20.485.0
---

> Support has been added for certificate authentication and for retrieving the JWT session token from the device certificate only via REST API and MQTT. For details, refer to [Device authentication](https://cumulocity.com/docs/device-integration/device-integration-rest/#device-authentication).

Reviewer 3: Cumulocity IoT now supports the ability to infer the full certificate chain from an intermediate certificate, enabling devices that are only able to send the device certificate to connect to Cumulocity IoT.  For full details refer to https://cumulocity.com/docs/device-integration/device-integration-rest/#device-authentication.

> Support has been added for certificate authentication and retrieving JWT session token from device certificate only via REST API and MQTT. For details, refer to the [Device authentication](https://cumulocity.com/docs/device-integration/device-integration-rest/#device-authentication) user documentation.

Reviewer: ```suggestion
Support has been added for certificate authentication and for retrieving the JWT session token from the device certificate only via REST API and MQTT. For details, refer to [Device authentication](https://cumulocity.com/docs/device-integration/device-integration-rest/#device-authentication).
```

> Support has been added for certificate authentication and retrieving JWT session token from device certificate only via REST and API and MQTT. For details, refer to the [Device authentication](https://cumulocity.com/docs/device-integration/device-integration-rest/#device-authentication) user documentation.

Reviewer 2: Trivial : `REST and API and MQTT` => `REST API and MQTT`

Author: Done
